pkgs/by-name/li/libnvidia-container/package.nix

```diff
@@ -14,44 +14,51 @@
   removeReferencesTo,
   replaceVars,
   go,
+  applyPatches,
 }:
 let
   modprobeVersion = "550.54.14";
-  nvidia-modprobe = fetchFromGitHub {
-    owner = "NVIDIA";
-    repo = "nvidia-modprobe";
-    rev = modprobeVersion;
-    sha256 = "sha256-iBRMkvOXacs/llTtvc/ZC5i/q9gc8lMuUHxMbu8A+Kg=";
-  };
-  modprobePatch = substituteAll {
-    src = ./modprobe.patch;
-    inherit modprobeVersion;
+  nvidia-modprobe = applyPatches {
+    src = fetchFromGitHub {
+      owner = "NVIDIA";
+      repo = "nvidia-modprobe";
+      rev = modprobeVersion;
+      sha256 = "sha256-iBRMkvOXacs/llTtvc/ZC5i/q9gc8lMuUHxMbu8A+Kg=";
+    };
+    patches = [
+      (substituteAll {
+        src = ./modprobe.patch;
+        inherit modprobeVersion;
+      })
+    ];
   };
 in
 stdenv.mkDerivation rec {
   pname = "libnvidia-container";
-  version = "1.16.2";
+  version = "1.17.2";
 
   src = fetchFromGitHub {
     owner = "NVIDIA";
     repo = "libnvidia-container";
     rev = "v${version}";
-    sha256 = "sha256-hX+2B+0kHiAC2lyo6kwe7DctPLJWgRdbhlc316OO3r8=";
+    sha256 = "sha256-JmJKvAOEPyjVx2Frd0tAMBjnAUTMpMh1KBt6wr5RRmk=";
```

Comment:
`hash` is updated. Sticked with `rev` instead of `tag`, since the latter doesn't seem to be passed through, so the `src.{tag,rev}` reference in `postPatch` didn't work anymore.